dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
+    return head :bad_request unless %w(word picture email).include? login_type
```

Comment:
[Yes](https://github.com/code-dot-org/code-dot-org/blob/ff04d9729831022f7b9c6c0f775687078f7f4a00/dashboard/app/models/sections/section.rb#L68). This really should be validated in the model.